assets/sass/05-blocks/quote/_style.scss

```
@@ -16,18 +16,40 @@
 		}
 	}
 
+	&::before {
+		color: currentColor;
+		content: "\201C";
+		display: block;
+		font-size: 3rem;
+		font-weight: 700;
+		line-height: 1;
+		position: absolute;
+		top: 4px;
+		left: -20px;
+	}
+
 	p {
+		color: var(--global--color-primary);
 		font-family: var(--quote--font-family);
 		font-size: var(--quote--font-size);
 		font-style: var(--quote--font-style);
 		line-height: var(--quote--line-height);
+		font-weight: 700;
 	}
 
 	.wp-block-quote__citation,
 	cite,
 	footer {
 		color: var(--global--color-primary);
-		font-size: var(--global--font-size-xs);
+		font-size: var(--global--font-size-sm);
+		font-style: var(--pullquote--font-style);
```

Did you mean for this to be `--quote--font-style`? Same issue in `_editor.scss`.
```suggestion
		font-style: var(--quote--font-style);
```